geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/conf/GeoMesaProperties.scala

```diff
@@ -0,0 +1,36 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.conf
+
+import java.util.Properties
+
+import com.typesafe.scalalogging.LazyLogging
+
+object GeoMesaProperties extends LazyLogging {
+
+  private val EmbeddedFile = "/org/locationtech/geomesa/geomesa.properties"
```

Comment:
no, but it currently only has the version from the pom - not something anyone should be modifying.